### DISCUSSION TITLE
能给Table的删除按钮弹出框内容添加一个编辑模板吗？

### DISCUSSION THREAD

**Discussion by yangf85** [dotnetcore/BootstrapBlazor#3255] [Q&A] [answered]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe the problem.

![image](https://github.com/dotnetcore/BootstrapBlazor/assets/42733580/84686d74-5978-4891-87e1-48281ecbd9b9)
现在文字多了以后，排版不是很清晰

### Describe the solution you'd like

如图

**Comment by bb-auto** [comment]
@yangf85 Thank you for reporting. We will give triage later.

**Comment by ArgoZhang** [comment]
@yangf85 可以，但是优先级不高，删除确认而已，你弄模板要干啥？

**Comment by yangf85** [comment]
> @yangf85 可以，但是优先级不高，删除确认而已，你弄模板要干啥？

看上去顺眼吧，其实也不是很必要

**Comment by ArgoZhang** [comment, marked as answer]
@yangf85 比较合理的设计应该是，如果文字比较多或者需要格式化，建议使用其他形式，比如弹窗提示 `Swal` `Dialog` 都可以，目前按钮是 `PopupConfirmButton` 不是为大段文字设计的
